bin/tictactoe

```diff
@@ -1,3 +1,9 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+puts "Welcome to Tic-Tac-Toe!" 
+
+game = Game.new
+
+game.start
```

Comment:
This executable is laudably terse, but it would be preferable to separate the interface into it's own class (e.g. GameCLI). This separates the execution logic from the implementation logic, so that you could add additional interfaces.